protocol/x/accountplus/keeper/keeper.go

```diff
@@ -53,7 +54,14 @@ func (k Keeper) GetAllAccountStates(ctx sdk.Context) ([]types.AccountState, erro
 
 	accounts := []types.AccountState{}
 	for ; iterator.Valid(); iterator.Next() {
-		accountState, found := k.GetAccountState(ctx, iterator.Key())
+		key := iterator.Key()
+
+		// Temporary workaround to exclude smart account kv pairs.
+		if strings.HasPrefix(string(key), types.SmartAccountKeyPrefix) {
+			continue
+		}
```

Comment:
skip these keys for now. eventually we might want to migrate these keys to be under some prefix 

lesson here is that we should always add some kind of prefix to keys to be more future proof